compiler/luci-interpreter/pal/linux/KernelsToBuild.lst

```diff
@@ -26,6 +26,7 @@ REGISTER_KERNEL(FullyConnected)
 REGISTER_KERNEL(Gather)
 REGISTER_KERNEL(Gelu)
 REGISTER_KERNEL(Greater)
+REGISTER_KERNEL(GRU)
 REGISTER_KERNEL(GreaterEqual)
```

Comment:
(Optional)
I think It looks good to apply alphabetically.
```suggestion
REGISTER_KERNEL(Greater)
REGISTER_KERNEL(GreaterEqual)
REGISTER_KERNEL(GRU)
```

Reply:
updated